src/error.rs

```diff
@@ -122,18 +122,84 @@ impl fmt::Display for ErrorType {
 #[derive(Debug, PartialEq, Deserialize)]
 #[serde(rename_all = "snake_case")]
 pub enum ErrorCode {
-    InvalidNumber,
-    InvalidExpiryMonth,
-    InvalidExpiryYear,
-    InvalidCvc,
-    InvalidSwipeData,
-    IncorrectNumber,
+    AccountAlreadyExists,
+    AccountCountryInvalidAddress,
+    AccountInvalid,
+    StripeAccount,
+    AccountNumberInvalid,
+    AlipayUpgradeRequired,
+    AmountTooLarge,
+    AmountTooSmall,
+    ApiKeyExpired,
+    BalanceInsufficient,
+    BankAccountExists,
+    BankAccountUnusable,
+    BankAccountUnverified,
+    BitcoinUpgradeRequired,
+    CardDeclined,
+    DeclineCode,
+    ChargeAlreadyCaptured,
+    ChargeAlreadyRefunded,
+    ChargeDisputed,
+    ChargeExpiredForCapture,
+    CountryUnsupported,
+    CouponExpired,
+    CustomerMaxSubscriptions,
+    EmailInvalid,
     ExpiredCard,
+    IncorrectAddress,
     IncorrectCvc,
+    IncorrectNumber,
     IncorrectZip,
-    CardDeclined,
-    Missing,
```

Comment:
The generic `missing` error code was removed here, but still exists in the Stripe documentation
https://stripe.com/docs/error-codes#missing

(Should be added after `LivemodeMismatch`?)